.github/workflows/erlang-tests.yml

```
@@ -0,0 +1,200 @@
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    env:
+      COUCHDB_URL: http://admin:admin@localhost:5984
+      INFLUXDB_URL: http://localhost:8086
+      INFLUXDB_TOKEN: admintoken
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)